### ISSUE TITLE
Sourcemaps are empty

### ISSUE THREAD

**Issue by at0g** [jtangelder/sass-loader#92]
When clicking on the line numbers to a scss file processed with sass-loader, the source file is empty.

The issue can be reproduced with a minimal config from the repo below:

https://github.com/at0g/webpack-css-sourcemaps

**Comment by jhnns** [comment]
I know that issue, but afaik it is not restricted to the sass-loader, it's a webpack-dev-server issue. I've experienced this with javascript code as well @sokra

**Comment by at0g** [comment]
Thanks for the quick response.
It seems odd that the sourcemaps from less-loader do work as expected if
the issue lies with webpack-dev-server

**Comment by sokra** [comment]
What should be wrong with the webpack-dev-server?

**Comment by jhnns** [comment]
I'll investigate this as soon as I've found some time.

**Comment by sairion** [comment]
I am expriencing same issue, but `.js` sourceMaps are served fine.

**Comment by atgillette** [comment]
I'm experiencing a similar problem without using webpack-dev-server.

**Comment by bestic** [comment]
I was able to turn on source map for sass by using "sourceMapContents" param:

```javascript
{
     test: /\.scss$/,
     loader: ExtractTextPlugin.extract('style-loader', 'css-loader?sourceMap!sass-loader?outputStyle=expanded&sourceMap=true&sourceMapContents=true')
}
```

**Comment by boopathi** [comment]
@bestic . Amazing ! Didn't know this option existed only. Damn.

**Comment by at0g** [comment]
Thanks @bestic - good workaround

**Comment by jhnns** [comment]
See https://github.com/jtangelder/sass-loader/pull/104#issuecomment-108525870

**Comment by jhnns** [comment]
Fixed with 1.0.4
